apps/full-site-editing/package.json

```diff
@@ -76,6 +77,7 @@
 		"@wordpress/url": "*",
 		"classnames": "2.2.6",
 		"lodash": "*",
-		"moment": "*"
+		"moment": "*",
+		"newspack-blocks": "github:Automattic/newspack-blocks#1.0.0-alpha.26"
```

Comment:
Note that we will handle updating to the latest version of the Newspack block in a separate PR.